.github/ISSUE_TEMPLATE/us_kafka_user_topic_request.yaml

```diff
@@ -0,0 +1,68 @@
+name: "User support: Kafka Topic/User request"
+description: Template for users to request Kafka Topics and User credentials/certs.
```

Comment:
Perhaps delete everything before "request" to make this read like other descriptions: "Request Kakfa topics and user credentials"